MakeHist/HTLT.C

```diff
@@ -98,4 +128,100 @@ void HTLT(){
 
     leg_HTLT->Draw();
 
+    c1->SaveAs("./hists/"+prefix+"_HTLT.png");
+
+
+    //make bkg for siginificance plot
+    HTLT_DB->Add(HTLT_ST);
+    HTLT_DB->Add(HTLT_TT);
+    HTLT_DB->Add(HTLT_DY);
+
+    TH1F * HTLT_DB_copy = (TH1F*) HTLT_DB->Clone("DB_copy");
+
+    TH1F * HTLT_200_copy = (TH1F*) HTLT_200->Clone("200_copy");
+
+    //std::cout << "ComputeIntegral " << HTLT_DB->ComputeIntegral() << "\n";
+    float previousBin;
+    float currentBin;
+    int nBins = HTLT_DB->GetNbinsX();
+
+    //std::cout << "right cut \n";
```

Comment:
Please describe the selection properly

Reply:
I don't understand this comment wrt to the highlight code. Can you say which selection you mean?

Reply:
I mean line 148. Please add here proper description of selection instead of "right cut"